packages/earth-map/src/components/collection/collection-downloadmetrics/CollectionDownloadMetrics.tsx

```diff
@@ -0,0 +1,198 @@
+          <div className="legend-item-group--radio ng-margin-top ng-margin-medium-bottom">
+            <div className="ng-display-inline-block ng-margin-medium-right">
+              <input
+                type="radio"
+                id={`radio-csv`}
+                value={'csv'}
+                name="fileType"
+                ref={register({
+                  required: true,
+                })}
+                className="marapp-qa-downloadmetricscsv"
+              />
+              <label htmlFor={`radio-csv`}>
+                <span className="legend-item-group--symbol" />
+                <span className="legend-item-group--name">CSV</span>
+              </label>
+            </div>
+            <div className="ng-display-inline-block ng-margin-medium-left">
+              <input
+                type="radio"
+                id={`radio-json`}
+                value={'json'}
+                name="fileType"
+                ref={register({
+                  required: true,
+                })}
+                className="marapp-qa-downloadmetricsjson"
+              />
+              <label htmlFor={`radio-json`}>
+                <span className="legend-item-group--symbol" />
+                <span className="legend-item-group--name">JSON</span>
+              </label>
+            </div>
+          </div>
+          <button
+            type="submit"
+            className="marapp-qa-actiondownload ng-button ng-button-primary ng-margin-right"
+            disabled={!isValid || isSubmitting || !dirty || !metricsWatcher?.length}
+          >
+            {t('Download')}
+          </button>
+          <button
+            className="marapp-qa-actioncancel ng-button ng-button-secondary"
+            onClick={onCancel}
+          >
+            {t('Cancel')}
+          </button>
+        </Card>
+      </div>
+    </form>
+  );
+
+  async function onSubmit(values) {
+    onDownloadStart();
+
+    const { metrics, fileType } = values;
+
+    try {
+      const data = await MetricService.downloadMetrics(collectionSlug, {
+        filter: serializeFilters({
+          slug: metrics.map((metric) => metric.value),
+        }),
+        group: organization,
+        include: 'location',
+        select: 'location.name',
+      });
+
+      const zip = new JSZip();
+
+      const metricTypes = groupBy(data, 'slug');
+
+      Object.keys(metricTypes).forEach((metricType) => {
+        const normalizedData = metricTypes[metricType].map((item) => ({
+          '#': item.location.name,
+          ...item.metric,
+        }));
+        const fileName = `${metricType}.${fileType}`;
+
+        if (fileType === 'csv') {
+          const json2csvParser = new json2csv.Parser();
+
+          zip.file(
+            fileName,
+            json2csvParser.parse(
+              normalizedData.map((item) => ({
+                '#': item['#'],
+                ...flatten(omit(item, '#')),
+              }))
```

Comment:
If you can find a valid constant name to replace '#' eg: `nr` the code would become:

```suggestion
normalizedData.map(({nr, ...item}) => ({
  nr,
  ...flatten(item),
}))
```